portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,36 @@
+.rectangle {
+  background-color:whitesmoke;
+  width: 1000px;
+  border: 10000px rgb(192, 255, 241);
+  padding: 50px;
+  margin: 10px;
+}
+
+.center {
+  text-align: center;
+  color: white;
+  font-size: 300%;
+}
+
+.centerText {
+    text-align: center;
+    color: white;
+    font: white;
+}
+
+.white {
+    color: white;
+    font-size: 200%
+}
+
+body {
+  background: url(images/fondo_topic2.jpg);
+  background-size: 100%;
+}
```

Comment:
I think we need to get together sometime to talk about how the website will look so that we can all be using the same style

Reply:
(but I think this is good for now)

Reply:
I agree with you, this style is just for now, but later we need to put together the same style for whole webapp:)